### DISCUSSION TITLE
Is there a way to keep the subtitle appearance after reopening IINA?

### DISCUSSION THREAD

**Discussion by ghost** [iina/iina#4175] [General]
Everytime I launch IINA, the subtitle appearance comes back to default style.

**Comment by michelklaasen** [comment]
You can change the default style in Settings > Subtitles (Cmd + ,) 😋

Although sometimes the subtitles seem to be fixed in style and cannot be changed I think that's cause the style is hardcoded for some of them.

**Reply by ghost** [reply]
thank you!
